src/containers/InputContainer.tsx

```diff
@@ -0,0 +1,100 @@
+import React, {useState} from 'react'
+import {v4 as uuidv4} from "uuid";
+import { useAppSelector, useAppDispatch } from '../hooks';
+import { addMessage } from '../state/messageSlice';
+import styled from 'styled-components';
+import { Client, Message } from '../state/types';
+
+const Container = styled.div`
+  width: 90vw;
+  height: 4rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 5vh;
+  z-index: 1;
+`
+
+const Form = styled.form`
+  width: 90vw;
+  max-width: 45rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`
+
+const Button = styled.button`
+  width: 4rem;
+  height: 4rem;
+  background-color: rgba(255, 255, 255, 0.5);
+  border-radius: 4rem;
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  &:hover {
+    box-shadow: 0 0 10px rgba(255, 255, 255, 0.6);
+    background-color: rgba(255, 255, 255, 0.8);
+  }
+  & p {
+    font-size: 2.5rem;
+    width: 3rem;
+    text-align: center;
+  }
+`
+
+const Input = styled.input`
+  height: 4rem;
+  width: 70%;
+  border-radius: 4rem;
+  border-color: transparent;
+  padding-left: 1.2rem;
+  outline: none;
+    box-shadow: 0px 0px 20px rgba(255, 255, 255, 1);
+  &:hover {
+    box-shadow: 0 0 10px rgba(255, 255, 255, 0.6);
+  }
+  font-size: 1rem;
+`
+function InputContainer() {
+  const [message, setMessage] = useState("");
+  
+  const {clients, currentClient} = useAppSelector((state)=>state.client);
```

Comment:
```suggestion
  const {dummy, currentClient} = useAppSelector((state)=>state.client);
```
clients는 사용되지 않음을 네이밍이나 주석을통해 알려주시면 좋을것 같습니다.

Reply:
향후 사용하려는 부분을 이와 같이 나타낼 수도 있겠군요.